apps/src/templates/button.module.scss

```diff
@@ -242,7 +243,8 @@ a:visited,
     }
 
     &:focus {
-      border-color: $brand_primary_default;
+      outline: $brand_primary_default solid 2px;
+      outline-offset: 2px;
```

Comment:
Nit and def not blocking: you could make a mixin for the focus state since it's repeatable 🤓

Reply:
Oh brilliant! And good mixin practice for me too.

Reply:
Okay I may or may not have copied exactly what you did in the pegasus styles ... but it's done! Good call –– thanks.